gateway-release/home/conf/topologies/homepage.xml

```diff
@@ -70,6 +75,20 @@
    <service>
       <role>KNOX-METADATA</role>
    </service>
+   <service>
+      <role>KNOXTOKEN</role>
+      <param>
+         <name>knox.token.ttl</name>
+         <value>604800000</value>
+      </param>
+      <param>
+         <name>knox.token.target.url</name>
+         <value>cdp-proxy-token/</value>
+      </param>
```

Comment:
nit: The target URL should have been removed

Reply:
I've removed the 'cdp-' to make more generic. Thanks for the catch.